README.md

```diff
@@ -1,72 +1,115 @@
+1. Test Environment: `api.qa.stellarstation.com:443`
+2. Production Environment: `api.stellarstation.com:443`
+
+We will use the Test Environment for the purpose of this walkthrough.
+
+Create an environment variable for your machine. This depends on your operating system and shell.
+
+Linux
+
+We will use Bash shell for the purpose of this walkthrough.
+
+1. Open ~/.bashrc
+1. Add `export STELLARSTATION_API_URL="api.qa.stellarstation.com:443"` to the end of the file.
+1. Save and close the file.
+1. In the terminal, execute `source ~/.bashrc`
+
+Windows
+
+It will look something like [this procedure](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7.4#set-environment-variables-in-the-system-control-panel).
+
+1. Search `Environment Variables` in the Windows search bar and open the `Environment Variables` manager.
+1. Create a new environment variable with the name `STELLARSTATION_API_URL` and the value `api.qa.stellarstation.com:443`.
+1. Log out and log back in.
+
+### Authentication
+
+You will need an API key to fully use the CLI. This walkthrough assumes you have created an account on the appropriate server and know how to generate an API key through the StellarStation Console. As of writing this, there are two possible StellarStation Consoles available for users to log in to:
+1. Test Environment: `www.qa.stellarstation.com`
+2. Production Environment: `www.stellarstation.com`
+
+Generate a private API key, download it, and place it somewhere safe.
+
+__IMPORTANT__: Do not share this private key with anyone.
+
+Execute `stellar auth activate-api-key path/to/key.json`.
+
+### Change the TLE
+
+Now that you have completed authentication, we can update the TLE. This walkthrough assumes an Infostellar support engineer has configured a test ground station for your account below 51 degrees latitude.
+
+Execute `stellar sat add-tle [STELLARSTATION SATELLITE ID HERE] "[TLE LINE 1]" "[TLE LINE 2]"`. Try the [Space Station](https://celestrak.org/NORAD/elements/gp.php?CATNR=25544) as a good temporary TLE for now.
+
+Now that you have provided a new TLE, wait several seconds, and check that the TLE propagated to StellarStation.
+
+Execute `stellar sat get-tle [STELLARSTATION SATELLITE ID HERE]`. The returned TLE should match the one you provided above.
+
+__WARNING__: `add-tle` sets the TLE updater in StellarStation to "Manual" mode - meaning that StellarStation will not automatically update the TLE. To set up automatic TLE updates please contact an Infostellar support engineer.
 
-```bash
-$ stellar -h
-```
+### Getting a list of Available Passes
 
-If you see anything is unclear, feel free to file an issue or send a pull request.
+We can see a list of available passes now that the TLE is set.
 
-## Authentication
+Execute `stellar sat list-passes [STELLARSTATION SATELLITE ID HERE] --verbose`.
```

Comment:
`stellar satellite list-passes`

Reply:
To clarify, you want me to expand the name of 'sat' to 'satellite'?